src/ui/tray-icon.cpp

```diff
@@ -352,13 +346,11 @@ QIcon SeafileTrayIcon::stateToIcon(TrayState state)
 #endif
 }
 
-void SeafileTrayIcon::toggleMainWindow()
+void SeafileTrayIcon::showMainWindow()
 {
     MainWindow *main_win = seafApplet->mainWindow();
     if (!main_win->isVisible()) {
```

Comment:
Fine. I thought it was a qt's method. If it is a qt's method instead of a home-brewed method, I'd rather to do some check here.

Done in 89fee0c